src/middleware.js

```diff
@@ -24,6 +13,16 @@ function skip(ignoreRouteUrls) {
     };
 }
 
+function asStream(logger) {
```

Comment:
I readded the stream function removed above. It's the most convenient way to make sure that the logs morgan generates receive a log-level.

The morgan documentation says it wants a stream passed to the middleware but in the original version of this code (with `addStream`) we didn't actually pass a stream and instead just wrapped the logger in an object with a `write` property because that is the api morgan actually requires. It worked for us in the past so I assume it won't cause problems.